applications/serial_lte_modem/doc/PPP_AT_commands.rst

```diff
@@ -93,3 +95,32 @@ Example
   #XPPP: 0,0
 
   OK
+
+Testing on Linux
+================
+
+You can test SLM's PPP on Linux by using the ``pppd`` command.
+
+For that, SLM must be compiled without CMUX because there is no widely available utility that allows multiplexing a device file on Linux.
+If you happen to be equipped with one, you can use SLM's PPP with ``pppd`` through CMUX by first setting up CMUX and replacing the device file in the ``pppd`` command by that of SLM's PPP channel (that will have been created by the CMUX utility).
+
+Assuming that SLM's UARTs are connected to the Linux host, you first need to get PPP running.
+For that, start SLM and issue an ``AT+CFUN=1`` command.
+Make sure that the network registration succeeds and that PPP is started successfully, either by having a look at SLM's logs or issue an ``AT#XPPP?`` command that should return ``#XPPP: 1,0``.
+Then, run the following command:
+
+   .. code-block:: console
+
+      $ sudo pppd -detach <PPP_UART_dev> <baud_rate> noauth crtscts novj nodeflate nobsdcomp debug +ipv6 usepeerdns noipdefault defaultroute defaultroute6 ipv6cp-restart 5 ipcp-restart 5
+
+Make sure to replace ``<PPP_UART_dev>`` by the device file assigned to the PPP UART (for example, :file:`/dev/ttyACM1`) and ``<baud_rate>`` by the baud rate of the UART that PPP is using.
+
+After you have run the command and the PPP link negotiation has completed successfully, a new network interface will be created (typically ``ppp0``)
+This network interface will send and receive IP traffic through the modem of the nRF91 Series SiP running SLM.
```

Comment:
```suggestion
After you have run the command and the PPP link negotiation has completed successfully, a new network interface will be created, typically ``ppp0``.
This network interface will send and receive IP traffic through the modem of the nRF91 Series SiP running SLM.

If you have an utility that allows multiplexing a device file on Linux, you can use SLM's PPP with the ``pppd`` command through CMUX.
To do this, you must set up CMUX, and then replace the device file in the ``pppd`` command with SLM's PPP channel, which will have been created by the CMUX utility.
```

Reply:
The CMUX part here is coming too late to me because if CMUX is used then it must be set up *before* PPP, so before running the `pppd` command.

Reply:
This could be something for a note before the steps, then. That way it's clear that it's something that can be done but modifies the noted steps.

Reply:
My thinking, though not clearly communicated, was that the part about CMUX would not be part of numbered steps, but rather content after them.

Reply:
Yeah, I was also thinking about making it a note before the numbered steps start.

Reply:
By the way, let me correct you for once: `a` utility, not `an`. :wink: